### PR TITLE
scripts to run montagu on different instances

### PR DESCRIPTION
**Pull request by hillalex**
Ok, one script to setup the instance, one to start, one to stop.

### CODE REVIEW

**Comment by hillalex** [comment]
Actually I guess we also want scripts to stop each of these. Ok ignore this, refactoring.

File: setup.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```
Comment (richfitz) [inline setup.sh]: could you make this file be called `setup` (no `.sh` suffix) & similar for the other scripts

File: start.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -ex
+
+instance=$(cat instance)
+
+if [[ "$instance" = "uat" ]] || [[ "$instance" = "science" ]] || [[ "$instance" = "prod" ]] ; then
+    orderly-web start ./config --extra $instance
+else
+    echo 'You must run ./setup.sh to configure the instance name'
+    exit 0
```
Comment (richfitz) [inline start.sh]: should be `exit 1` to indicate failure

File: setup.sh [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+if [[ $# -ne 1 ]] ; then
+    echo 'Usage: ./setup.sh INSTANCE_NAME'
+    exit 0
```
Comment (richfitz) [inline setup.sh]: should be exit 1 throughout

File: start.sh [hunk 4]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -ex
```
Comment (richfitz) [inline start.sh]: can you drop the `x` option to clear up the output? (and in the others)

File: setup.sh [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+fi
+
+if [[ "$1" = "uat" ]] || [[ "$1" = "science" ]] || [[ "$1" = "prod" ]] ; then
```
Comment (richfitz) [inline setup.sh]: `prod` should be `production` to match our names elsewhere I think

**Review by richfitz — CHANGES_REQUESTED** [review]

**Review by richfitz — APPROVED** [review]

**Review by EmmaLRussell — APPROVED** [review]

**Pull request merged**